src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -610,93 +605,99 @@ object DataPointsResource {
     }
 
   def parseStringDataPoints(response: DataPointListResponse): Seq[StringDataPointsByIdResponse] =
-    response.items
+    response.getItemsList.asScala
       .map(x =>
         StringDataPointsByIdResponse(
-          x.id,
-          optionalString(x.externalId),
-          x.isString,
-          optionalString(x.unit),
-          x.getStringDatapoints.datapoints
-            .map(s => StringDataPoint(Instant.ofEpochMilli(s.timestamp), s.value))
+          x.getId,
+          optionalString(x.getExternalId),
+          x.getIsString,
+          optionalString(x.getUnit),
+          x.getStringDatapoints.getDatapointsList.asScala
+            .map(s => StringDataPoint(Instant.ofEpochMilli(s.getTimestamp), s.getValue))
+            .toSeq
         )
       )
+      .toSeq
```

Comment:
Apparently these (and the rest of them) are not needed here

Reply:
IntelliJ is using 2.12, where they weren't/aren't necessary, but with the revamped collections in Scala 2.13 (and Scala 3, as it's using the same standard library as Scala 2.12) they are necessary, because we get a `scala.collection.mutable.Buffer` here, but "normal" `Seq` is immutable in 2.13: so `.toSeq` converts from a mutable buffer to an immutable `Seq`-like thing.

I'll add some comments, as IntelliJ will mark this as unused until we change the ordering of Scala versions in build.sbt.